src/foam/u2/images/round-check-circle-green.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
```

Comment:
so I've looked this up and actually found - we currently have 4 green check-marks - all the same ... :(
/foam/u2/images/checkmark-small-green.svg
/net/nanopay/images/checkmark-large-green.svg
/net/nanopay/images/checkmark-small-green.svg
/foam/nanos/controller/checkmark-small-green.svg

![image](https://user-images.githubusercontent.com/10802216/83922495-63179900-a74e-11ea-866d-f086ee978e8d.png)

Reply:
haha oh wow, ok i'll remove the others thanks for checking that